libtenzir/builtins/operators/every_cron.cpp

```diff
@@ -194,20 +206,23 @@ class every_operator final : public operator_base {
   }
 
   auto name() const -> std::string override {
-    return "every";
+    return std::string{scheduler_.name};
   }
 
-  friend auto inspect(auto& f, every_operator& x) -> bool {
-    return f.object(x).fields(f.field("op", x.op_),
-                              f.field("interval", x.interval_));
+  friend auto inspect(auto& f, scheduled_execution_operator& x) -> bool {
+    return f.object(x).fields(f.field("op", x.op_), x.scheduler_.field(f));
```

Comment:
This works, but the way we usually do this is as follows:
```suggestion
  friend auto inspect(auto& f, scheduled_execution_operator& x) -> bool {
    return f.object(x).fields(f.field("op", x.op_), f.field("scheduler", scheduler_));
```

Then you define an `inspect` friend function for the scheduler.

Reply:
Awesome that you think about such things! However, we do not use CAF serialization for persistance. Hence we are free to change `inspect` whenever we want.